app/models/user.rb

```diff
@@ -1,9 +1,12 @@
 class User < ApplicationRecord
-  has_many :comments
-  has_many :likes
-  has_many :posts
-  validates :photo, :bio, presence: true
-  validates :posts_counter, numericality: { only_integer: true }, comparison: { greater_than_or_equal_to: 0 }
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Please make sure that the app is confirmable and ask for confirmation by email, I can't see confirmable for the user in the devise right now.